pkg/cmd/addon/addon.go

```diff
@@ -354,6 +362,24 @@ func (o *addonCmdOpts) validate() error {
 	return nil
 }
 
+func checkBreakingChange(o *addonCmdOpts) error {
+	if o.addonEnableFlags.noChange() && o.addon.Status.Phase == extensionsv1alpha1.AddonEnabled {
```

Comment:
what if enable and enabled addon, and disable an disabled addon?  For changes that wont modify Addon CR, there won't be a 'PATCH'.

Reply:
![image](https://github.com/user-attachments/assets/7da2bcf9-9af0-4ae4-aa4e-247ecfd0141e)

it will do patch, but change nothing, even if the scenario like kb08->09 and addon08